README.md

```diff
@@ -2,13 +2,24 @@
 
 **A CSS stylesheet to quickly highlight a11y concerns.**
 
-An alternative testing method to Chrome's Lighthouse.
+An alternative testing method for accessibility.
+
+Designed to quickly highlight some common accessibility errors and warnings that can be easily and quickly rectified.
+
+Recommended to use as a first line of defence in accessibility testing, before turning to automation tools and scanners such as Lighthouse on Chrome.
+
+Remember not to push this to production!
```

Comment:
Yeah, I added this to the 'Other' project board (can you see the boards?)

How I'd do this, is in a `.env.dev` file, set a property to true, such as `INCLUDE_CHECKA11Y = true`, then have `INCLUDE_CHECKA11Y = false` is files such as `.env.staging` and `.env/production`. But this is a very PWA approach to me, ish.

Reply:
Yeah this will need more fleshing out. Maybe somewhere down the line we can provide easy integration guides with raw/jekyll/wp/react/vue/etc.

Very long term scope though. Focus on MVP for now and assume users will know how to manage this themselves.

Reply:
Sure! 😊